git-bash-hints.qmd

```diff
@@ -0,0 +1,188 @@
+---
+title: "Git Bash Hints"
+editor: visual
+---
+
+This page will cover a few useful commands that you can use in Git Bash. 
+
+:::{.callout-note}
+
+You will see many generalised paths, directory names, file names, etc. on this page. For example, you will see something like `cd /path_to/working_directory/you_want` where you would need to replace the `/path_to/working_directory/you_want` part with a path that is applicable to your work.
+
+:::
+
+## Checking and Navigating Directories
+
+The commands in this section allow you to check and change your current working directory, moving between different folders in your file system.
+
+`pwd`: This stands for "print working directory", so it will show you your current working directory for Git.
+
+You can use it by entering the following in the terminal: 
+
+```{r, eval = FALSE}
```

Comment:
These should be switched to bash rather than r, although I'm not sure it will add any extra rendering off the back of that. Also, I don't think you need the comma.

```suggestion
```{bash eval = FALSE}
```